exporters/logging-otlp/src/test/java/io/opentelemetry/exporter/logging/otlp/OtlpJsonLoggingLogRecordExporterTest.java

```diff
@@ -5,177 +5,47 @@
-                  .setVersion("1")
-                  .setAttributes(Attributes.builder().put("key", "value").build())
-                  .build())
-          .setBody("body1")
-          .setSeverity(Severity.INFO)
-          .setSeverityText("INFO")
-          .setTimestamp(100L, TimeUnit.NANOSECONDS)
-          .setObservedTimestamp(200L, TimeUnit.NANOSECONDS)
-          .setAttributes(Attributes.of(stringKey("animal"), "cat", longKey("lives"), 9L))
-          .setSpanContext(
-              SpanContext.create(
-                  "12345678876543211234567887654322",
-                  "8765432112345876",
-                  TraceFlags.getDefault(),
-                  TraceState.getDefault()))
-          .build();
-
-  private static final LogRecordData LOG2 =
-      TestLogRecordData.builder()
-          .setResource(RESOURCE)
-          .setInstrumentationScopeInfo(
-              InstrumentationScopeInfo.builder("instrumentation2").setVersion("2").build())
-          .setBody("body2")
-          .setSeverity(Severity.INFO)
-          .setSeverityText("INFO")
-          .setTimestamp(100L, TimeUnit.NANOSECONDS)
-          .setObservedTimestamp(200L, TimeUnit.NANOSECONDS)
-          .setAttributes(Attributes.of(booleanKey("important"), true))
-          .setSpanContext(
-              SpanContext.create(
-                  "12345678876543211234567887654322",
-                  "8765432112345875",
-                  TraceFlags.getDefault(),
-                  TraceState.getDefault()))
-          .build();
-
-  @RegisterExtension
-  LogCapturer logs = LogCapturer.create().captureForType(OtlpJsonLoggingLogRecordExporter.class);
-
-  LogRecordExporter exporter;
-
-  @BeforeEach
-  void setUp() {
-    exporter = OtlpJsonLoggingLogRecordExporter.create();
+import java.io.OutputStream;
+import javax.annotation.Nullable;
+
+@SuppressLogger(LogRecordExporter.class)
+class OtlpJsonLoggingLogRecordExporterTest
+    extends AbstractOtlpJsonLoggingExporterTest<OtlpJsonLoggingLogRecordExporter> {
+
+  public OtlpJsonLoggingLogRecordExporterTest() {
+    super(
+        "logging-otlp",
+        TestDataExporter.forLogs(),
+        OtlpJsonLoggingLogRecordExporter.class,
+        ConfigurableLogRecordExporterProvider.class,
+        "expected-logs.json",
+        "expected-logs-wrapper.json",
+        "OtlpJsonLoggingLogRecordExporter{wrapperJsonObject=false, jsonWriter=LoggerJsonWriter}");
   }
 
-  @Test
-  void log() throws Exception {
-    exporter.export(Arrays.asList(LOG1, LOG2));
+  @Override
+  protected OtlpJsonLoggingLogRecordExporter createExporter(
+      @Nullable OutputStream outputStream, boolean wrapperJsonObject) {
+    OtlpJsonLoggingLogRecordExporterBuilder builder =
+        OtlpJsonLoggingLogRecordExporterBuilder.create();
+    if (outputStream != null) {
+      builder.setOutputStream(outputStream);
+    }
+    return builder.setWrapperJsonObject(wrapperJsonObject).build();
```

Comment:
I want to avoid the code duplication in methods such as 

```java
 @Test
  void shutdown() {
    assertThat(exporter.shutdown().isSuccess()).isTrue();
    assertThat(
            exporter
                .export(Collections.singletonList(METRIC1))
                .join(10, TimeUnit.SECONDS)
                .isSuccess())
        .isFalse();
    assertThat(logs.getEvents()).isEmpty();
    assertThat(exporter.shutdown().isSuccess()).isTrue();
    logs.assertContains("Calling shutdown() multiple times.");
  }
```

Reply:
ok, I get it now. Since `OtlpJsonLoggingLogRecordExporter` doesn't have a builder, we don't need the variations
I'll revert the test then